### DISCUSSION TITLE
Consider supporting 6mm belts

### DISCUSSION THREAD

**Discussion by dflemstr** [stevenbennett/DIYson#24] [Ideas]
It seems to be way easier to find 6mm belts compared to 5mm belts, and it also seems to fit fine into all the aluminum extrusions I had laying around.

**Comment by legopc** [comment]
Yeah I agree. I ordered 5mm belts just to be sure but I might also order some 6mm to compare. 6mm does fit, is easier to find locally so would be a good replacement for the partslist.

**Comment by stevenbennett** [comment]
@dflemstr @legopc I'll keep it in mind. The nice thing about the 5mm belts is that as far as I know you can often find them where v-rollers and rails are sold. Though I appreciate that outside of that scenario that that 6mm belts are easier to find. Thanks for flagging that.

The 6mm belt will fit on paper, but there's a chance it will rub agains the aluminum rails. If either of you want to test out a 6mm belt let me know and I can send you the Fusion 360 files for the relevant parts and you can give it a go.

**Comment by legopc** [comment]
Haha that's funny because I think @dflemstr and I have the exact opposite with regards to the availability of the belt. When I started looking for parts I often found them on 3d printing sites and they all sell 6mm. 

I have the lamp (apart from the LED related stuff) almost complete but still waiting on the 5mm belt but I'll order some 6mm tonight and see how that fits and report back. I don't have them yet so I am not sure but I would imagine the belt doesn't have to wobble much side to side, and it fits "in" the profile just fine so I don't see much of an issue there. 

I would love the Fusion 360 files to tinker.

**Reply by stevenbennett** [reply]
@legopc Two questions for you if you're comfortable answering: roughly where are you located and where did you purchase the v-rails and v-wheels?

**Reply by legopc** [reply]
I'm located in The Netherlands. I bought everything from Aliexpress because I wanted to consolidate sellers as much as possible and I want it to be as cheap as possible. 
Those kinds of parts are mostly found at 3d printer webshops as 3d printer replacement parts, seeing as v-rails, wheels, pully and washers are all used on 3d printers. For the change I made I just looked up "openbuilds *nameofrelevantpart*" and for example came to one of the Polish sites that happens to ship all over Europe and has a lot of parts meaning easy consolidation of sellers. 
By the way if you are asking this because you want to change or update the BOM, if you give me some constraints and requirements, I'll update it for you. I would also like a manual of some sorts of what should go where according to you because I think there are some redundant parts in the current BOM.

**Reply by stevenbennett** [reply]
@legopc Cool thanks for those details. So did you end up finding a suitable 5mm belt on Aliexpress by chance?

**Comment by dflemstr** [comment]
+1, I have not been able to find any 5mm belt anywhere except importing from China (aliexpress etc).  I have made a cutting jig to shave off 1mm from a 6mm belt.  Maybe worth mentioning that I'm in Europe, not in the US.

The idler pulley in the BOM is also for 6mm belt, and I haven't been able to find a 5mm pulley for the same reasons as above.

I would really appreciate some STLs for 6mm parts to try out!  I guess I can also take some Fusion 360 files, but Fusion 360 tends to be a laggy broken mess for me :)

**Reply by stevenbennett** [reply]
@dflemstr Thanks for letting me know, I didn't know 5mm belts were uncommon in Europe so I'll keep that in mind going forward. Out of curiosity, where would you go to buy v-rails and v-wheels if you were building a DIYson?

**Reply by dflemstr** [reply]
Personally I used a OpenBuilds retailer like RatRig (https://ratrig.com/hardware2/openbuilds.html) that carries the exact parts in the BOM, but I know there are definitely cheaper suppliers too like @legopc mentioned.  It's nice to only have to pay the shipping costs once and I happened to need a bunch of other stuff from RatRig at the time, which is why I chose them.

EDIT: what's sort of funny is I think they very recently (~a week or two ago? or my google-fu might be failing me) started listing 5mm belt but it isn't the right pitch: https://ratrig.com/hardware2/openbuilds/openbuilds-gt3-timing-belt-1mt.html.  The pitch of course only matters for attaching the ends of the belt in this build, and doesn't really matter for the pulley.

**Reply by stevenbennett** [reply]
@dflemstr Good to know. Nice catch on the belt pitch, that would have be easy to miss. All of this info is helpful because like you and @legopc I really want to minimize and simplify sources as much as possible to keep shipping costs low. Thanks for the info 👍🏻

**Comment by Bindall** [comment]
In the UK we have  'ooznest.co.uk' they have 5mm belts and are partnered with open builds . After a quick browse they appear to have all the open builds parts available. Granted UK - Europe trade is not quite as smooth sailing as it once was . Hope that helps

**Reply by stevenbennett** [reply]
Nice thanks for this

**Comment by dflemstr** [comment]
@stevenbennett since you uploaded the Base f3d files, is there any chance you can also upload the files for this thread so we can try out 6mm belt?

**Reply by stevenbennett** [reply]
Yup I will upload it for you. I'm procrastinating a bit because I have to fix an error with the CAD file that I've been putting off. But it's on my list :)

**Comment by legopc** [comment]
I have not tried reworking the models, also because my printer is broken, but nah the 6mm belt is not it. It *just* rubs against everything. Unlike the 5mm belt, the 6mm one just barely doesn't fit in-between the slots in the profile. Its either in or out of the profile. Some things like the pully mount could be fixed I think but the belt meets the frame again pretty quickly.

**Reply by stevenbennett** [reply]
I really appreciate you taking the time to test this out and report back. This was my suspicion (the belt rubbing against the interior profile) but I couldn't get around to actually experimenting with it so I could never say for sure one way or the other. Thanks for giving it a try, your testing will likely save people some time and frustration 🙂

**Reply by dflemstr** [reply]
I have a very different experience, since I have been using 6mm belt all this time (but trimmed off 0.5mm from each side just near the attachment points).  For me, 6mm works absolutely fine with no issues!  This should also be the case, since standard V-slot aluminum profiles should have a 6mm gap, and the V-slot means that there is a lot more space where the belt actually ends up traveling (maybe ~8mm), and the belt is resting in the "riverbed" that the profile provides, without touching the profile.

Maybe having a 6mm belt would cause issues for some, if they are using non-standard profiles or maybe T-slot instead of V-slot, but I'm still eagerly awaiting the 6mm models (or at this point I might start hacking the .stls myself) because it works great in general!
